generic_config_updater/services_validator.py

```diff
@@ -104,7 +104,7 @@ def caclmgrd_validator(old_config, upd_config, keys):
 
 
 def ntp_validator(old_config, upd_config, keys):
-    return _service_restart("ntp-config")
+    return _service_restart("chrony")
```

Comment:
Where are `old_config` and `upd_config` being used? As far as I can tell, all that the test case is doing is restarting chrony without changing the configuration.